### ISSUE TITLE
choosing states (unit of analysis variables)

### ISSUE THREAD

**Issue by katevcoulter** [MIDS-at-Duke/unifying-data-science-final-project-primaries#1]
treatment states: minnesota, idaho, colorado, maine
- we are currently missing final data at the aggregate level for maine but have the other 3
control states: ??????
- let's figure this out monday at our meeting (should all be states that didn't change their primary structure for dem pres elections and be similar demographically to our treatment states)

*** will also assign shota when he joins this

**Comment by katevcoulter** [comment]
added shota

**Comment by aberman6** [comment]
I just did a very messy first pass at the d&d analysis. I came out with a couple of questions.

1. Which states make sense to include as our treatment group?
2. Are there any states we should exclude from the analysis all-together?
3. Are we only including states that have complete data from 2008-2020? If so this is very limiting.

**Comment by katevcoulter** [comment]
1. treatment = our original 4 states (Minnesota, Idaho, Maine, Colorado)

**Comment by katevcoulter** [comment]
2. exclude = needs more work, but we are 100% excluding Hawaii, Alaska, California, Ohio and any other state affected by covid-19 thus having their primary moved

**Comment by katevcoulter** [comment]
3. not necessarily but also depends on which vars are missing

**Comment by katevcoulter** [comment]
i'm going through the data now and adding a new treatment column to mark the ones we've assigned to treated, control or exclude if it's blank then we still should double check to decide if we should include or not

**Comment by katevcoulter** [comment]
shoot i meant wisconsin not ohio my b

**Comment by aberman6** [comment]
I updated the cleaning_2.R file and the turnout_v3.csv with new exclusion criteria. I excluded:
- States that are strange like Cali
- States whos primaries were effected by COVID
- States that didn't have Democrat turnout or VEP numbers for 2008, 2016, and 2020.

That leaves us with 25 states (4 treatment states and 21 control states).

_**Control** = Alabama, Arizona, Arkansas, Florida, Iowa, Massachusetts, Michigan, Mississippi, Missouri, Nevada, New Hampshire, North Carolina, North Dakota, Oklahoma, South Carolina, Tennessee, Texas, Utah, Vermont, Virginia, & Washington_

_**Treatment** = Colorado, Idaho, Maine, & Minnesota_
